MainTest.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class MainTest{
+
+  @Test 
+
+
+public void testNothing(){
+
+    
+}
+
+
+
+@Test
+public void testSomeStuff(){
+
+  int a = 2;
+  int b = 3;
+  
+  assertEquals("1. test failed",(a + b) ,5);
```

Comment:
Laut JUnit4 Doku: 
assertEquals(String message, long expected, long actual)
          Asserts that two longs are equal.

ist der erste Parameter der MessageString, der zweite, das was erwartet wird und der dritte Parameter der Aufruf der zu prüfenden Methode. 
Deshalb hätte ich hier `assertEquals( "1. test failed", 5, (a + b) );` erwartet